mixer/pkg/il/interpreter/interpreterRun.got

```diff
@@ -635,6 +635,13 @@ func (in *Interpreter) run(fn *il.Function, bag attribute.Bag, step bool) (Resul
 			tu64 = math.Float64bits(tf64)
 			STACK_PUSH2(uint32(tu64>>32), uint32(tu64&0xFFFFFFFF))
 
+		case il.SizeS:
```

Comment:
Actually, can you add stack underflow guard test for this as well?
It should be TestInterpreter_Eval_StackUnderflow

Reply:
Will do in a followup.